examples/manual_routes/src/tests.rs

```diff
@@ -0,0 +1,80 @@
+use super::*;
+use rocket::{Rocket, Route, Catcher};
+use rocket::testing::MockRequest;
+use rocket::http::{ContentType, Status};
+use rocket::http::Method::*;
+
+fn route_and_mount() -> Rocket{
```

Comment:
I think it's a bad idea to rewrite all of the ignition + mounting code here since then we need to change two places. Can you put all of this into a `rocket` function in `main.rs` and then call that function from here and in `main`? See `examples/state` for an example.